sound/soc/sof/intel/hda-dai.c

```diff
@@ -414,6 +414,16 @@ static struct snd_soc_cdai_ops sof_probe_compr_ops = {
 #endif
 #endif
 
+#define SND_SOC_PLAYBACK_STREAM(_name, _min, _max) .playback = \
+	{.stream_name = _name, .channels_min = _min, .channels_max = _max,},
+
+#define SND_SOC_CAPTURE_STREAM(_name, _min, _max) .capture = \
+	{.stream_name = _name, .channels_min = _min, .channels_max = _max,},
+
+#define SND_SOC_STREAM(_pname, _cname, _min, _max) \
+	SND_SOC_PLAYBACK_STREAM(_pname, _min, _max) \
+	SND_SOC_CAPTURE_STREAM(_cname, _min, _max)
```

Comment:
and then here
```
+#define SND_SOC_STREAM(_pname, _cname, _min, _max) \
+	SND_SOC_PLAYBACK_STREAM(_pname, _min, _max), \
+	SND_SOC_CAPTURE_STREAM(_cname, _min, _max)
```

Reply:
@bardliao these have got to go in a header so other drivers can use it too no?

Reply:
+1